src/app.html

```diff
@@ -346,32 +414,26 @@ <h5 class="header col s12 light">My Resume. Available in various formats, but if
         <div class="section">
 
             <div class="collection">
-                <a title="Archimedes Trajano Resume in Word Format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.docx"><i class="fa fa-file-word-o left"> </i> Word</a>
-                <a title="Archimedes Trajano Resume in PDF format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.pdf"><i class="fa fa-file-pdf-o left"> </i> PDF</a>
-                <a title="Archimedes Trajano Resume" class="collection-item waves-effect waves-light" href="https://trajano.net/archimedes-trajano-resume/"><i class="fa fa-eye left"> </i> View online</a>
-                <a class="waves-effect waves-light collection-item" href="http://stackoverflow.com/users/story/242042"><i class="fa fa-stack-overflow left"> </i>  Stack Overflow Developer Story</a>
-                <a class="waves-effect waves-light collection-item" href="https://linkedin.com/in/trajano"><i class="fa fa-linkedin left"> </i>  LinkedIn Profile</a>
+                <a title="Archimedes Trajano Resume in Word Format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.docx"><i class="fa fa-file-word-o fa-fw"> </i> Word</a>
+                <a title="Archimedes Trajano Resume in PDF format" class="collection-item waves-effect waves-light " href="assets/Archimedes%20Trajano.pdf"><i class="fa fa-file-pdf-o fa-fw"> </i> PDF</a>
+                <a title="Archimedes Trajano Resume" class="collection-item waves-effect waves-light" href="https://trajano.net/archimedes-trajano-resume/"><i class="fa fa-eye fa-fw"> </i> View online</a>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)